crates/storage/libmdbx-rs/src/environment.rs

```diff
@@ -579,6 +580,8 @@ pub struct EnvironmentBuilder {
     /// The maximum duration of a read transaction. If [None], but the `read-tx-timeout` feature is
     /// enabled, the default value of [DEFAULT_MAX_READ_TRANSACTION_DURATION] is used.
     max_read_transaction_duration: Option<read_transactions::MaxReadTransactionDuration>,
+    /// MDBX exclusive flag.
+    exclusive: bool,
```

Comment:
I don't think that we need to add this field here, because it's already in `flags: EnvironmentFlags`